tools/euclideandistance_to_destination.py

```diff
@@ -0,0 +1,75 @@
+from math import sqrt
+from entities import Shipment, Region
+from config import Config
+import numpy as np
+from environment import Environment
+
+# Calculates the distance between a shipment pickup location and the shipment destination
+
+def euclideandistance_to_destination(env : Environment, object1 : Shipment):
```

Comment:
Write one generic Euclidean distance method, and call that instead of repeating yourself here several times.